CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ if(WIN32 AND MSVC_VERSION LESS 1800)
 endif()
 
 option(ENABLE_JSON_LOGGING "Adds additional JSON debug logging to the response" OFF)
+option(DEBUG_GEOMETRY "Enables an option to dump GeoJSON of the final routing graph" OFF)
```

Comment:
I'll make a branch then remove this from this PR.  It depends on the edge_segment lookup loop in this PR, so it can't go into develop before.